.github/workflows/release.yml

```diff
@@ -275,19 +275,19 @@ jobs:
 
           # Remove backslashes before colons in commit details
           new_fix_notes=$(echo "${new_fix_notes}" | sed 's/\\:/:/g')
-          
-          # Combine existing content with new fix notes
```

Comment:
- Ensure that the variables are used consistently and correctly throughout the script.